tests/clickhouse-test

```diff
@@ -73,13 +73,7 @@ def stringhash(s):
 
 # First and last lines of the log
 def trim_for_log(s):
-    if not s:
-        return s
-    lines = s.splitlines()
-    if len(lines) > 100:
-        return "\n".join(lines[:50] + ["#" * 100] + lines[-50:])
-    else:
-        return "\n".join(lines)
```

Comment:
Why?

Reply:
Because it may hide very useful debug output (and I experienced this issue when debugging the flakiness of `02435_rollback_cancelled_queries`). And because I don't really care about the size of the log (until it gets enormous and grep becomes slow). But maybe it would be better to change the limits to a few thousand lines.

Reply:
Also because `############` scares me :)